docs/book/basics/project_creation_and_structure.md

```diff
@@ -1,3 +1,7 @@
+{% panel style="info", title="Under Development" %}
+This book is being actively developed.
+{% endpanel %}
```

Comment:
There has to be a better way of sticking a panel on top of very page :)

Reply:
Probably :P

Reply:
I'll do some research next week.  Being able to apply headers to all pages would be useful generally.